src/app/layout.tsx

```diff
@@ -31,6 +32,7 @@ export default async function RootLayout({
   return (
     <html lang="en" className="h-full" suppressHydrationWarning>
       <body className="flex min-h-full bg-white antialiased dark:bg-zinc-900">
+        <GoogleAnalytics gaId="G-LN41LVPXY2" />
```

Comment:
lets introduce a env_example with `NEXT_PUBLIC_GA_ID` ... we will need to set this in he deploy as well